### ISSUE TITLE
Tabs focus order on mobile

### ISSUE THREAD

**Issue by steverep** [home-assistant/frontend#14117]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

When viewing the settings sub-pages on mobile, the navigation tabs are moved to the bottom of the screen, but the focus order (i.e. tab key order) remains the same as on desktop or larger screen.  This is visually confusing and inconsistent with nearly all other mobile apps.

### Describe the behavior you expected

Either the navigation tabs should be at the top as on desktop, or the focus order should change such that the tabs are last.

### Steps to reproduce the issue

1. Go to Settings then Devices and Services (for example)
2. Use any means to demo the focus order on mobile, such as text-to-speech, screen reader, or simulator on desktop browser.
3. Focus will start at back button and possibly search field, jump down to tabs,  then back to the top.

### What version of Home Assistant Core has the issue?

2022.10.x

**Comment by steverep** [comment]
Still valid issue

**Comment by karwosts** [comment]
If we fix this by changing the sort order so that the tabs are last, that may require tabbing through thousands of entities before you can reach the navigation bar at the bottom. Is that an acceptible design?

Was going to fix this but just noticed that behavior and it seems a little iffy to me, but I don't really use tabbing so I'm not sure how someone who does use it would feel about that.
